src/Options/docs.js

```diff
@@ -67,6 +68,7 @@
  * @property {String} restAPIKey Key for REST calls
  * @property {Boolean} revokeSessionOnPasswordReset When a user changes their password, either through the reset password email or while logged in, all sessions are revoked if this is true. Set to false if you don't want to revoke sessions.
  * @property {Boolean} scheduledPush Configuration for push scheduling, defaults to false.
+ * @property {SchemaOptions} schema Rest representation on Parse.Schema https://docs.parseplatform.org/rest/guide/#adding-a-schema
```

Comment:
Why does this PR add SchemaOptions?

Reply:
no idea aahaha

Reply:
Please remove if this is not auto-generated by `npm run definitions`. There is an issue with SchemaOptions https://github.com/parse-community/parse-server/issues/7805 which is a separate issue from this PR.